src/Facebook/FacebookClient.php

```diff
@@ -76,9 +76,9 @@ class FacebookClient
     protected $enableBetaMode = false;
 
     /**
-     * @var FacebookHttpClientInterface HTTP client handler.
+     * @var HttpClient HTTP client handler.
      */
-    protected $httpClientHandler;
+    protected $httpClient;
```

Comment:
Thanks @Ocramius! `master` is for the v6 major version, so this is OK for this PR.
About `private`, I totally agree, that's why we have #447